web/src/components/Project.vue

```diff
@@ -23,26 +30,39 @@ import ProjectRepository from '@/repositories/ProjectRepository'
 export default {
   name: 'Project',
   props: {
-    project: String
+    project: String,
   },
   data() {
     return {
       logoURL: '',
       latestVersion: '',
       versions: [],
-      hideAvatar: false
+      hideAvatar: false,
+      isFavourite: false,
     }
   },
   async created() {
     document.title = this.project + " | docat"
     this.logoURL = ProjectRepository.getProjectLogoURL(this.project)
     this.versions = await ProjectRepository.getVersions(this.project)
     this.latestVersion = (this.versions.find((version) => version.name == 'latest') || this.versions[0]).name
+    this.isFavourite = ProjectRepository.isFavourite(this.project)
+  },
+  methods: {
+    switchFavourite() {
+      ProjectRepository.setFavourite(this.project, !this.isFavourite);
+      this.isFavourite = !this.isFavourite;
+    }
   }
 }
 </script>
 <style lang="scss">
 .md-avatar.hidden {
   display: none;
 }
+#favourite-star {
+  color: yellow;
```

Comment:
I would go for `color: #fff;` and change the logic (bordered means not favourited, filled means favourited)

Reply:
Not sure I understand you here
You want yellow with #ffff00 or white with #fff ?
You would have a border only for not-favorites?

Reply:
Ah sorry I mean white #fff or #ffffff :slightly_smiling_face: 
Yes, so only border for non-favourites and fully filled out for favourites.